### PR TITLE
Fixes in response to Joakim Andén's review

### PR DESCRIPTION
**Pull request by MuawizChaudhary**
Currently several tests are broken. ~~these can be fixed just by reverting the lines where I added the startswith() function~~ (Edit: Just take the negation...). The skcuda 1d backend seems to have been broken by me in the process of fixing things, don't remember if its due to the usage of the startswith function, I will fix all these and more in the morning.

### CODE REVIEW

**Comment by MuawizChaudhary** [comment]
Oddly, the test's fail when cuda is capitalized to CUDA here: https://github.com/MuawizChaudhary/kymatio/blob/documentation/kymatio/scattering2d/backend/torch_skcuda_backend.py#L219. Seems like very unexpected behavior. Thoughts @edouardoyallon ?

**Comment by edouardoyallon** [comment]
@MuawizChaudhary so it's simply because there is a test that tests the error message.. basically it just tests that a pattern or a string is present, and I guess it's sensitive to caps... I followed a standard practice(apparently) but it might be a bit unneficient

File: kymatio/scattering1d/backend/torch_skcuda_backend.py [hunk 1]
```diff
@@ -62,15 +62,17 @@ def __call__(self, x):
...
-            raise RuntimeError('Input should be contiguous.')
-
+        
         if not is_complex(x):
             raise TypeError('The input and outputs should be complex.')
 
+        if not x.is_contiguous():
+            warnings.warn("Modulus: tensor x is converted to a contiguous
+                    array.")
+            x = x.contiguous()
```
Comment (MuawizChaudhary) [inline kymatio/scattering1d/backend/torch_skcuda_backend.py]: Let me me know if these changes are not needed.

Comment (edouardoyallon) [reply kymatio/scattering1d/backend/torch_skcuda_backend.py]: That's actually great.

**Pull request merged**